examples/SPI/D_Ne_SPI_ITER.py

```diff
@@ -99,10 +100,39 @@
 B0 = 5.3            # magnetic field strength in Tesla
 
 # Set up radial grid
+R0 = 6.2
+kappa = 1.2
+
+ds.radialgrid.setType(RGrid.TYPE_ANALYTIC_TOROIDAL)
```

Comment:
While it doesn't appear to be a problem for this particular setup, keep in mind that if solving a kinetic equation in toroidal geometry one should really use a appropriately tuned pitch grid (i.e. use ``MomentumGrid.setTrappedPassingBoundaryLayerGrid()`` in the Python settings interface)